src/DotLiquid.Tests/Tags/StandardTagTests.cs

```diff
@@ -701,5 +709,59 @@ public void TestLegacyKeyValueDrop()
             Assert.AreEqual("Green", drop.BeforeMethod("Last"));
             Assert.IsNull(drop.BeforeMethod("UnknownProperty"));
         }
+

```

Comment:
Add test cases incrementing and decrementing two different values to ensure incrementing or decrementing one counter does not increment or decrement another comment